base_business_document_import/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016-2019 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base Business Document Import",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "license": "AGPL-3",
+    "summary": "Provides technical tools to import sale orders or supplier invoices",
+    "author": "Akretion, Nicolas JEUDY, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": [
+        # odoo
+        "account",
+        # OCA/partner-contact
+        "base_vat_sanitized",
```

Comment:
Please remove `base_vat_sanitized` from the dependencies.
See also https://github.com/OCA/partner-contact/pull/821#issuecomment-593145213

and

https://github.com/OCA/partner-contact/blob/13.0/base_vat_sanitized/readme/DESCRIPTION.rst

Reply:
Thanks, the dependency has been removed and the code updated